pkg/bucketaccessrequest/bucketaccessrequest.go

```diff
@@ -57,13 +58,24 @@ func (b *bucketAccessRequestListener) Add(ctx context.Context, obj *v1alpha1.Buc
 	return nil
 }
 
+// Update is called in response to a change to BucketAccessRequesst. At this point
+// BucketAccess cannot be changed once created as the Provisioner might have already acted upon the create BucketAccess and created the backend Bucket Credentials
+// Changes to Protocol, Provisioner, BucketInstanceName, BucketRequest cannot be allowed. Best is to delete and recreate a new BucketAccessRequest
+// Changes to ServiceAccount, PolicyActionsConfigMapData and Parameters should be considered in lieu with sidecar implementation
 func (b *bucketAccessRequestListener) Update(ctx context.Context, old, new *v1alpha1.BucketAccessRequest) error {
 	glog.V(1).Infof("Update called for BucketAccessRequest %v", old.Name)
```

Comment:
Many of these logs only show the BAR name but omit its namespace. Thus the logged name may not be unique.

Reply:
I see no change to reflect ns in log msgs.

Reply:
I fixed it by using ns/name format.